shared/database/source_control/file.py

```diff
@@ -149,6 +149,13 @@ def video_parent_file(self, session):
     task_id = Column(Integer, ForeignKey('task.id'))
     task = relationship("Task", foreign_keys = [task_id])
 
+    # Connection ID where the file is stored (if no connection provided will use default storage provider from env vars)
+    connection_id = Column(Integer, ForeignKey('connection_base.id'))
+    connection = relationship("Connection", foreign_keys = [connection_id])
+    bucket_name = Column(String())
+
+    storage_strategy = Column(String(), default="global_env_vars")   # ['connection', 'global_env_vars'] 
```

Comment:
@PJEstrada  I just added [this](https://github.com/diffgram/diffgram/commit/a30a4f0cf87b1d045b67d24e1d0fd105fea6d4f0) as an idea for your review... also not sure on naming I offered `default` in other case, but maybe it's better to define it directly like `global_env_vars`

Reply:
Hey Anthony! I like more the first approach since we really enfoce we have all the data needed for actual url generation. In this case we might need have cases where storage_strategy might be one thing but does not have connection_name or bucket. I think the function gives us better guarantees for that if you're ok with it :)